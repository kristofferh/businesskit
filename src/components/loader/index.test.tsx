import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";
import { Loader } from "./index";

describe("Loader", () => {
  it("renders and matches the snapshot", () => {
    const tree = renderer.create(<Loader size={30} color="#ee99ee" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders the component", () => {
    render(<Loader />);
  });
});
