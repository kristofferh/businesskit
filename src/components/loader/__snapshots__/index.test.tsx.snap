// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders and matches the snapshot 1`] = `
@keyframes animation-0 {
  from {
    -webkit-transform: rotate(0deg);
    -moz-transform: rotate(0deg);
    -ms-transform: rotate(0deg);
    transform: rotate(0deg);
  }

  to {
    -webkit-transform: rotate(360deg);
    -moz-transform: rotate(360deg);
    -ms-transform: rotate(360deg);
    transform: rotate(360deg);
  }
}

.emotion-2 {
  position: relative;
  width: 30px;
  height: 30px;
}

.emotion-1 {
  transform-origin: center center;
  -webkit-animation: animation-0 0.9s infinite linear;
  animation: animation-0 0.9s infinite linear;
  display: block;
  pointer-events: none;
}

.emotion-0 {
  stroke-dashoffset: 75;
  stroke-dasharray: 100;
  stroke-linecap: round;
}

<div
  className="emotion-2"
>
  <svg
    className="emotion-1"
    height={30}
    viewBox="0 0 30 30"
    width={30}
  >
    <g
      fill="none"
    >
      <circle
        className="emotion-0"
        cx="50%"
        cy="50%"
        r={12.5}
        stroke="#ee99ee"
        strokeWidth={5}
      />
    </g>
  </svg>
</div>
`;
